torch/_subclasses/fake_tensor.py

```diff
@@ -358,17 +374,18 @@ def new(self, *args, **kwargs):
         # so in order to use the same pattern as normal invocation of
         # returning meta device within the kernel we need to intercept
         # the call here
+        # because it doesn't go through the dispatcher, we run into errors
+        # when attempting to compute an output in meta, so
+        # we compute the real tensor then convert to meta
```

Comment:
But this means that `new` is no longer memory efficient, is that right?

Reply:
Yea, this is sort of a deprecated API / doesn't go through the dispatcher, so it would make sense that it might be fragile to changes. Similar to fallbacks, seemed okay to allocate a real tensor temporarily.

Reply:
OK. It's just odd that although we have explicit handling for it, we aren't able to just go ahead and write the "correct" fake rule for it. But I suppose it might be annoying to do.